operations/rates.md

```diff
@@ -116,9 +226,11 @@ Note this operation uses [Pagination](../guidelines/pagination.md) and supports
 | `IsActive` | boolean | required | Whether the rate is still active. |
 | `IsEnabled` | boolean | required | Whether the rate is currently available to customers. |
 | `IsPublic` | boolean | required | Whether the rate is publicly available. |
+| `Type` | [Rate type](rates.md#rate-type) | required | Type of the rate |
 | `Name` | string | required | Name of the rate. |
 | `ShortName` | string | required | Short name of the rate. |
```

Comment:
```suggestion
| `ShortName` | string | required | Short name of the rate (in the first listed language). |
```

Reply:
I've checked the code and it takes default language. I've adjusted your description.